modules/account-lib/src/coin/avaxc/keyPair.ts

```diff
@@ -0,0 +1,41 @@
+import { KeyPairOptions, DefaultKeys, isPrivateKey, isPublicKey } from '../baseCoin/iface';
+import { NotImplementedError } from '../baseCoin/errors';
+import { Secp256k1ExtendedKeyPair } from '../baseCoin/secp256k1ExtendedKeyPair';
+
+export class KeyPair extends Secp256k1ExtendedKeyPair {
+  /**
+   * Public constructor. By default, creates a key pair with a random master seed.
+   *
+   * @param { KeyPairOptions } source Either a master seed, a private key, or a public key
+   */
+  constructor(source?: KeyPairOptions) {
+    super(source);
+    if (source) {
+      if (isPrivateKey(source)) {
+        super.recordKeysFromPrivateKey(source.prv);
+      } else if (isPublicKey(source)) {
+        super.recordKeysFromPublicKey(source.pub);
+      } else {
+        throw new Error('Invalid key pair options');
+      }
+    } else {
+      throw new Error('Invalid key pair options');
+    }
```

Comment:
I think the constructor `implementation` and the `jsdoc` are not consistent within each other.

The doc says: `By default, creates a key pair with a random master seed`.
While the constructor throws an error if it's called without an argument.

I think the correct behavior is the one detailed on the jsdoc, but since this class will be implemented as part of [STLX-5078](https://bitgoinc.atlassian.net/browse/STLX-5078) maybe we can just add a comment like: `TODO(STLX-5078): Implement random seed and key pair generation`.